jdi-light-angular-tests/src/test/java/io/github/epam/TestsInit.java

```diff
@@ -0,0 +1,38 @@
+package io.github.epam;
+
+
+import com.epam.jdi.light.settings.WebSettings;
+import io.github.com.StaticSite;
+import io.github.epam.testng.TestNGListener;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Listeners;
+
+import static com.epam.jdi.light.driver.WebDriverUtils.killAllSeleniumDrivers;
+import static com.epam.jdi.light.driver.get.DriverData.DRIVER_NAME;
+import static com.epam.jdi.light.elements.composite.WebPage.openUrl;
+import static com.epam.jdi.light.elements.init.PageFactory.initSite;
+import static com.epam.jdi.light.settings.WebSettings.DOMAIN;
+import static com.epam.jdi.light.settings.WebSettings.logger;
+
+
+@Listeners(TestNGListener.class)
+public class TestsInit {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'TestsInit' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5408219)